lib/experimental-default-theme.json

```diff
@@ -129,7 +129,8 @@
 				"dropCap": true
 			},
 			"color": {
-				"custom": true
+				"custom": true,
+				"link": false
```

Comment:
maybe there's a better path here?

Reply:
I'm happy to approve as it is. In its current shape, the other option I can think of is making link a top-level element of `features`, which doesn't feel great either.

I also wonder if we should do a follow-up PR when we revisit/consolidate the `features` shape before releasing Gutenberg 9.0. I believe is the first release we include this feature and it'll be good to avoid changing shape later. Left some comments at https://github.com/WordPress/gutenberg/issues/20588#issuecomment-689425319

Reply:
We definitely need a follow-up PR to reshape the paths. That said, I'm not sure if it's before or after Gutenberg 9.0. The important thing is to do it before we make theme.json stable.

Reply:
It's not the first release for these things, 8.9 include one or two features and dropCap is there for a long time.